src/lib/containers/CertificationQuiz.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useEffect, useState } from 'react'
+import { SynapseClient } from '../utils'
+import { useSynapseContext } from '../utils/SynapseContext'
+import {
+  PassingRecord,
+  QuizResponse,
+  Quiz,
+} from '../utils/synapseTypes/CertificationQuiz/Quiz'
+import {
+  MULTICHOICE_RESPONSE_CONCRETE_TYPE_VALUE,
+  QuestionResponse,
+} from '../utils/synapseTypes/CertificationQuiz/Questions'
+import parse from 'html-react-parser'
+import { displayToast } from './ToastMessage'
+import { Button } from 'react-bootstrap'
+import { MarkdownPopover } from './MarkdownPopover'
+import { HelpOutlineTwoTone } from '@material-ui/icons'
+import Typography from '../utils/typography/Typography'
+
+const CertificationQuiz: React.FunctionComponent = () => {
+  const { accessToken } = useSynapseContext()
+  const [quiz, setQuiz] = useState<Quiz | undefined>()
+  const [questionResponse, setQuestionResponse] = useState<QuestionResponse[]>(
+    [],
+  )
+  const [passingRecord, setPassingRecord] = useState<PassingRecord>()
+
+  const gettingStartedUrl =
+    'https://help.synapse.org/docs/Getting-Started.2055471150.html'
+
+  const getQuiz = async () => {
+    try {
+      setQuiz(await SynapseClient.getCertifyQuiz(accessToken))
+    } catch (err: any) {
+      console.log(err)
+    }
+  }
+
+  useEffect(() => {
+    if (accessToken) {
+      getQuiz()
+    }
+  }, [accessToken])
+
+  useEffect(() => {
+    if (!accessToken) {
+      new Error('Please sign in to take certification quiz')
+    }
+  }, [accessToken])
```

Comment:
The error isn't thrown. Also, the call to the certifiedUserTest endpoint throws a 403 if the user isn't signed in. Could we just do this? -

1. make the API call regardless of whether or not the user has a token
2. Then pass the error to the error boundary either with ReactQuery's `useErrorBoundary: true` or `react-error-boundary`'s useErrorHandler().handleError.
3. The error boundary will render the appropriate alert